gcp/compute_engine/cloudrun_serverless_neg_https_lb/main.tf

```diff
@@ -0,0 +1,23 @@
+resource google_compute_region_network_endpoint_group serverless_neg {
+  project = var.project
+  region = var.region
+  name = var.serverless_neg_name
+  network_endpoint_type = "SERVERLESS"
+
+  cloud_run {
```

Comment:
Is cloud run the only option?

Reply:
there could be blocks for either cloudRun, appEngine or cloudFunction if we want to